src/mysky/index.ts

```diff
@@ -0,0 +1,160 @@
+export type { CustomConnectorOptions } from "./connector";
+export { DacLibrary } from "./dac";
+
+import { PermCategory, PermHidden, Permission, PermType, PermWrite } from "skynet-interface-utils";
+
+import { Connector, CustomConnectorOptions } from "./connector";
+import { SkynetClient } from "../client";
+import { DacLibrary } from "./dac";
+import { RegistryEntry, setSignedEntry, SignedRegistryEntry } from "../registry";
+import { CustomGetJSONOptions, CustomSetJSONOptions, getOrCreateRegistryEntry, JsonData } from "../skydb";
+import { hexToUint8Array } from "../utils/string";
+import { Signature } from "../crypto";
+import { deriveDiscoverableTweak } from "./tweak";
+
+export async function loadMySky(
+  this: SkynetClient,
+  appPath: string,
+  customOptions?: CustomConnectorOptions
+): Promise<MySky> {
+  const mySky = await MySky.init(this, customOptions);
+  mySky.addPermissions(new Permission(appPath, PermHidden, PermWrite));
+
+  return mySky;
+}
+
+export const mySkyDomain = "skynet-mysky.hns";
+
+export class MySky {
+  // ============
+  // Constructors
+  // ============
+
+  constructor(protected connector: Connector, protected permissions: Permission[]) {}
+
+  static async init(client: SkynetClient, customOptions?: CustomConnectorOptions): Promise<MySky> {
```

Comment:
If you have a static function that initializes your class, you usually want to mark the constructor as private to prevent developer error.

`init` is not a good name for something like this, because it implies it's initializing an instance of a certain class. I think `New` is better. 

Maybe this should be a singleton? 🤔  
you'd have an extra prop `public static instance: MySky` and only create an instance if that is undefined